support/ci/build_component.ps1

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env powershell
+
+#Requires -Version 5
+
+param (
+    # The name of the component to be built. Defaults to none
+    [string]$Component
+)
+
+# Since we are only verifying we don't have build failures, make everything
+# temp!
+$env:HAB_ORIGIN="throwaway"
+# let's make a selfcontained tempdir for this job
+$job_temp_root = mkdir (Join-Path $env:TEMP ([System.IO.Path]::GetRandomFileName()))
+$env:HAB_CACHE_KEY_PATH="$job_temp_root/keys"
+
+dir
+
+Write-Host "--- :key: Generating fake origin key"
+hab origin key generate
+Write-Host "--- :hab: Running hab pkg build for $Component"
+# REMOVE -R AFTER 0.75.0
+hab pkg build -D -R components/$Component
```

Comment:
should we just run all of these in `default-windows-privileged` and then change the above to `hab pkg build components/$Component`

Reply:
I'd rather run these in the single use instances for the time being - I ran into issues with the mounts and overlapping builds due to multiple agents on the same host.

Reply:
ok cool. In that case assuming we are using hab 0.75.0 or greater, the `-R` is not necessary

Reply:
Yep - I have a comment to that effect and I'm going to update our base hab image shortly